ddev/ddev
# feat: change upgrade pester interval to 4 hours, fixes #5889

## The Issue

* #5889 

People are running old versions of DDEV, and they say that they don't get properly notified about that.

## How This PR Solves The Issue

Pester them on 4-hour cycle instead of one-week cycle.

Here's the updated view of it (with the correct URL):

<img width="814" alt="image" src="https://github.com/ddev/ddev/assets/112444/d48904cb-cd5e-4cd5-8ef6-3969a5c2d1d7">


## Manual Testing Instructions

Try using an out-of-date version. You should get notified every 4 hours.

You can change the time on ~/.ddev/.update to force this cycle. `touch -d "5 hours ago" ~/.ddev/.update` or `touch -m -t 202402230500 ~/.ddev/.update`

## Automated Testing Overview



## Related Issue Link(s)

## Release/Deployment Notes
